A.java

```diff
@@ -1,3 +1,5 @@
 class A{
 
+int a;
+int b;
```

Comment:
What is the purpose of this parameter b.

Reply:
This will be used to store the value of breadth.